src/middlewares/renderLandingPage.js

```diff
@@ -9,9 +9,9 @@ import UrlShorten from '../models/UrlShorten';
  */
 export const renderLandingPage = (req, res) => {
 	UrlShorten.find({
-    created_by: req.cookies.userID //Find all clips created by this user.
+    createdBy: req.sessionID //Find all clips created by this user.
```

Comment:
I want to believe that the session must have been saved before calling this req.sessionId?

Reply:
can you explain it as a comment in the code?

Reply:
yeah, I have to fix a few things. the sessionId is read-only, set by node when the session is loaded. I am still wrapping my head around all this but according to @Stacalpha, we need to use cookies instead because they'll persist much longer. do not merge this commit yet

Reply:
oh great. I love your straight answer.